axlearn/cloud/gcp/system_characteristics.py

```diff
@@ -379,6 +379,28 @@ class _SystemCharacteristics:
         AcceleratorType["TPU"],
         "v5litepod-256",
     ),
+    # v6e
+    "v6e-4": _SystemCharacteristics(
+        "2x2", 1, "tpu-v6e-slice", "ct6e-standard-4t", 4, AcceleratorType["TPU"], "v6e-4"
+    ),
+    "v6e-8": _SystemCharacteristics(
```

Comment:
Seeing 
```
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.17.3/pkg/internal/controller/controller.go:227
2024-12-10T22:58:28Z	ERROR	Reconciler error	{"controller": "jobset", "controllerGroup": "jobset.x-k8s.io", "controllerKind": "JobSet", "JobSet": {"name":"ethanli-imagenet-resnet-resnet-2024121014","namespace":"default"}, "namespace": "default", "name": "ethanli-imagenet-resnet-resnet-2024121014", "reconcileID": "946de27a-c3e8-4f44-ab5c-8e377e71e763", "error": "job \"ethanli-imagenet-resnet-resnet-2024121014-job-0\" creation failed with error: admission webhook \"warden-validating.common-webhooks.networking.gke.io\" denied the request: GKE Warden rejected the request because it violates one or more constraints.\nViolations details: {\"[denied by tpu-accelerator-topology-constraints]\":[\"Invalid resource requests for google.com/tpu. It should be equal to one of [8] for accelerator tpu-v6e-slice and topology 2x4. Please request all the tpu chips available on the node.\"]}\nRequested by user: 'system:serviceaccount:jobset-system:jobset-controller-manager', groups: 'system:serviceaccounts,system:serviceaccounts:jobset-system,system:authenticated'."}
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
```
 in jobset controller